index.js

```diff
@@ -1,48 +1,73 @@
-/* eslint-disable */
-let library = [];
-
-function getInput() {
-  const book = {};
-  book.title = document.getElementById('bookTitle').value;
-  book.author = document.getElementById('bookAuthor').value;
-  return book;
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = Math.random();
+  }
+}
+
+class Library {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook(book) {
+    this.data.concat(book);
+    localStorage.setItem('library', JSON.stringify(this.data));
+    Display.addToUI(book);
+  }
+
+  removeBook(id) {
+    const book = document.getElementById(id);
+    book.remove();
+    this.data = this.data.filter((bookObj) => bookObj.id !== id);
+    localStorage.setItem('library', JSON.stringify(this.data));
+  }
 }
 
-function removeBook(title) {
-  const book = document.getElementById(title);
-  book.remove();
-  library = library.filter((bookObj) => bookObj.title !== title);
-  localStorage.setItem('library', JSON.stringify(library));
+const library = new Library();
+
+class Input {
+  static getInput() {
+    const title = document.getElementById('bookTitle');
+    const author = document.getElementById('bookAuthor');
+    const book = new Book(title.value, author.value);
+    title.value = '';
+    author.value = '';
+    return book;
+  }
 }
 
-function addBook(bookObj) {
-  const bookList = document.getElementById('book-list');
-  const book = document.createElement('LI');
-  book.setAttribute('id', bookObj.title);
-  book.innerHTML = `<h3> ${bookObj.title} </h3> <p>${bookObj.author} </p>`;
-  const deleteBtn = document.createElement('button');
-  deleteBtn.innerHTML = 'Delete';
-  deleteBtn.addEventListener('click', () => removeBook(bookObj.title));
-  book.appendChild(deleteBtn);
-  bookList.appendChild(book);
+class Display {
+  static addToUI(bookObj) {
+    const bookList = document.getElementById('book-list');
+    const book = document.createElement('li');
+    book.setAttribute('id', bookObj.id);
+    book.innerHTML = `<h3>"${bookObj.title}" by ${bookObj.author}</h3>`;
+    const deleteBtn = document.createElement('button');
+    deleteBtn.classList.add('remove');
+    deleteBtn.innerHTML = 'Remove';
+    deleteBtn.addEventListener('click', () => library.removeBook(bookObj.id));
+    book.appendChild(deleteBtn);
+    bookList.appendChild(book);
+  }
 }
 
 const addButton = document.getElementById('add-btn');
 addButton.addEventListener('click', () => {
```

Comment:
[OPTIONAL] I think you could select the form instead and listen for the submit event. This event not only triggers when the button it's clicked but also with the enter key. This results in a better UX experience. If you decide to follow my advice remember to change the type of the button to `submit` and prevent the default behavior of the form when it's submitted. [You can read more about it here.](https://betterprogramming.pub/click-vs-submit-eventlisteners-536b62be9359)